sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/CoverageResultImporter.java

```diff
@@ -47,45 +48,33 @@
 import org.sonar.plugins.php.phpunit.xml.PackageNode;
 import org.sonar.plugins.php.phpunit.xml.ProjectNode;
 
-public class PhpUnitCoverageResultImporter implements PhpUnitImporter {
+public class CoverageResultImporter extends SingleFileReportImporter {
 
-  private static final Logger LOG = LoggerFactory.getLogger(PhpUnitCoverageResultImporter.class);
+  private static final Logger LOG = LoggerFactory.getLogger(CoverageResultImporter.class);
 
-  private final FileSystem fileSystem;
-
-  protected Metric<Integer> linesToCoverMetric = CoreMetrics.LINES_TO_COVER;
-
-  protected Metric<Integer> uncoveredLinesMetric = CoreMetrics.UNCOVERED_LINES;
+  private static final String WRONG_LINE_EXCEPTION_MESSAGE = "Line with number %s doesn't belong to file %s";
 
-  protected CoverageType coverageType = CoverageType.UNIT;
+  protected Metric<Integer> linesToCoverMetric;
+  protected Metric<Integer> uncoveredLinesMetric;
+  protected CoverageType coverageType;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "CoverageType"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)